pkg/metrics/metrics_test.go

```diff
@@ -22,7 +22,10 @@ import (
 	"testing"
 	"time"
 
+	"github.com/jetstack/cert-manager/pkg/apis/certmanager/v1alpha1"
```

Comment:
nit: could you put the `cert-manager` internal imports into their own block? i.e.:

```
import (
   "stdlib-imports"

   "external-imports"

   "cert-manager imports"
)
```

We follow this convention to make it a bit easier to read 😄